nautobot/extras/datasources/git.py

```diff
@@ -177,7 +177,10 @@ def ensure_git_repository(repository_record, logger=None, head=None):  # pylint:
     if logger:
         if changed:
             logger.info("Repository successfully refreshed")
-        logger.info(f'The current Git repository hash is "{repository_record.current_head}"')
+        logger.info(
+            f'The current Git repository hash is "{repository_record.current_head}"',
+            extra={"object": repository_record},
```

Comment:
Thanks for the review!

The requested change is certainly doable but it would lead to losing the "linkability" of the object. If we leave it as-is, what do you expect to happen in the situation you described?

Reply:
I *think* in that case the `extra` just gets silently discarded without issue, but I'd want to test it to be certain.

Reply:
OK, I'll try find some time and test this tomorrow. If this proves to be true, what's our stance? Keep the repo name both in the log message and as an extra/object?

Reply:
Just tested this and indeed it's silently dropped. So the question remains: should we keep the repo name both in the log message and as an extra/object?

Reply:
Given the dual usage, I think it'd make sense to do both.

Reply:
Agreed and done. Thanks!